eZ/Publish/Core/FieldType/RelationList/Type.php

```diff
@@ -110,6 +114,28 @@ public function validateFieldSettings($fieldSettings)
                         );
                     }
                     break;
+                case 'selectionLimit':
```

Comment:
> Should the value be validated anywhere in field type? Or you only intend to validate in repo forms?

@andrerom AFAICS it's validalidated here, in `RelationList\Type`. Should we consider more places in ezpublish-kernel for validation?

Reply:
This is for the setting, was implying making sure we don't accept more than $selectionLimit number of destinationContentIds.